### PR TITLE
Feature/issue 133 polls settings

### PR DESCRIPTION
**Pull request by codiebeulaine**
@miltontony ready for review

### CODE REVIEW

File: polls/tests/test_voting.py [hunk 1]
```diff
@@ -96,3 +96,55 @@ def test_voting_once_only(self):
...
+                    {'choice': choice1.id})
+
+        client.login(username='tester2', password='tester2')
+        client.post(reverse('molo.polls:vote',
+                    kwargs={'question_id': question.id}),
+                    {'choice': choice1.id})
+        response = client.get(reverse(
+            'molo.polls:results',
+            kwargs={'poll_id': question.id}))
+        self.assertContains(response, '2')
```
Comment (KaitCrawford) [inline polls/tests/test_voting.py]: isn't there something more descriptive we could use here? this will pass if there's a '2' anywhere in the page

**Comment by KaitCrawford** [comment]
after the queries I raised :+1:

**Comment by codiebeulaine** [comment]
@KaitCrawford ready for another review

File: tuneme/templates/polls/results.html [hunk 2]
```diff
@@ -11,20 +11,33 @@
...
+						{% if question.result_as_percentage %}
+			        <span style="width:{{ choice.percentage }}%"></span>
+							{{ choice.percentage }}%
+						{% else %}
+							<span style="width:{{ choice.percentage }}%"></span>
+							{% if choice.votes = 1%}
+								{{ choice.votes }} vote
+							{% else %}
+								{{ choice.votes }} votes
+							{% endif %}
```
Comment (KaitCrawford) [inline tuneme/templates/polls/results.html]: there's a template tag for this:
https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#pluralize

**Comment by KaitCrawford** [comment]
:+1:

File: polls/models.py [hunk 3]
```diff
@@ -11,13 +11,30 @@
...
+            " instead of a percentage.")
+    )
     content_panels = Page.content_panels + [MultiFieldPanel([
-        FieldPanel('randomise_options')], heading="Question Settings",)]
+        FieldPanel('show_results'),
+        FieldPanel('randomise_options'),
+        FieldPanel('result_as_percentage')], heading="Question Settings",)]
+
+    def user_choice(self, user):
+        self.choicevote_set.filter(user=user)
```
Comment (miltontony) [inline polls/models.py:35]: @codiebeulaine this is unused.

Comment (codiebeulaine) [inline polls/models.py:35]: @miltontony it is used, in the template tags

**Pull request merged**